.github/workflows/lts-test.yaml

```diff
@@ -0,0 +1,37 @@
+on:
```

Comment:
When does this run?

Reply:
It runs when a pull request of cloud-opensource-java repository touches files matching the paths below (line 3)